### PR TITLE
Feature/pdf report

### PR DESCRIPTION
**Pull request by LoginALee**
Add functionality for generating a pdf report with users books

### CODE REVIEW

**Review by GregVaz — APPROVED** [review]
Great :rocket:

File: routes/books.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 var express = require("express");
 var router = express.Router();
+var path = require("path");
```
Comment (jvargas98) [inline routes/books.js]: I think you can change this for const

**Review by jvargas98 — APPROVED** [review]
🚀

**Review by ese-varo — APPROVED** [review]
Well done Ale :rocket: 
Btw, Happy Birthay! :birthday: :partying_face:

**Pull request merged**